### PR TITLE
[Shop] Bring back regular SyliusShop

### LINKED ISSUE #101: Conflict with SyliusShopBundle

### PR DESCRIPTION
**Pull request by lchrusciel**
Based on #114 

Resolves #101

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,13 @@ This repository provides a ShopApi implementation on the top of [Sylius E-Commer
                 stateless:  true
                 anonymous:  true
     ```
+    
+    5. Adjust checkout configuration to not colidate with Sylius shop
```
Comment (Arminek) [inline README.md]: `Sylius shop API`

Comment (venyii) [inline README.md]: `colidate` -> `colide`

Comment (pamil) [inline README.md]: `colide` -> `collide`? 🎉

Comment (Arminek) [inline README.md]: :tada:

Comment (lchrusciel) [inline README.md]: Thanks, guys :)

**Comment by pjedrzejewski** [comment]
@lchrusciel I think the regexp in Sylius/Sylius should be updated, WDYT?

**Comment by lchrusciel** [comment]
@pjedrzejewski I was thinking about that too. But on the other hand, if one would use different security config it is highly probable, that it will break the whole website, as the `sylius.security.shop_regex` parameter will not exist. Maybe, I will open a PR and then we will decide.

**Pull request merged**